<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeWindow</class>
 <widget class="QMainWindow" name="ThemeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="themebutton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="themebutton2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="themebutton3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="themebutton4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
